pyproject.toml

```diff
@@ -27,8 +27,8 @@ pytest = "7.2.1"
 pytest-dotenv = "0.5.2"
 pytest_httpserver = "1.0.8"
 pytest-mock = "3.11.1"
-typing-inspect = "0.8.0"
-typing_extensions = "^4.5.0"
+typing-inspect = "0.9.0"
```

Comment:
why this change

Reply:
I have the understanding with these Python types that it is good to keep them up to date and that they are entirely backward compatible. I could be wrong! I come from mostly a TS background and try to use types for everything.

Reply:
I can roll this back if it will cause issues